.github/workflows/publish.yaml

```
@@ -24,22 +24,39 @@ jobs:
         with:
           strip_v: true
 
+  get-previous-tag:
+    runs-on: ubuntu-22.04
+    outputs:
+      previous-tag: ${{ steps.previous-tag.outputs.tag }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Get previous tag
+        id: previous-tag
+        run: |
+          git fetch -a
+          tags=($(git tag --sort="-committerdate"))
+          previous_tag=${tags[1]}
+          echo "::set-output name=tag::$previous_tag"
+
   generate-release-notes-pr:
     runs-on: ubuntu-22.04
-    needs: [get-tag]
+    needs: [get-tag, get-previous-tag]
     if: github.ref_type != 'branch'
     steps:
     - name: Checkout
       uses: actions/checkout@v3
 
     - name: Generate Release Notes PR
       env:
+        GIT_PREV_TAG: ${{ needs.get-previous-tag.outputs.previous-tag }}
         GIT_TAG: ${{ needs.get-tag.outputs.tag }}
         GH_PAT: ${{ secrets.GH_PAT }}
       run: |
         curl -H "Authorization: token $GH_PAT" \
           -H 'Accept: application/json' \
-          -d "{\"event_type\": \"replicated-sdk-release-notes\", \"client_payload\": {\"version\": \"${GIT_TAG}\" }}" \
+          -d "{\"event_type\": \"replicated-sdk-release-notes\", \"client_payload\": {\"version\": \"${GIT_TAG}\", \"base\": \"${GIT_PREV_TAG}\" }}" \
```

i would recommend naming the param `prev_version` instead of `base`
```suggestion
          -d "{\"event_type\": \"replicated-sdk-release-notes\", \"client_payload\": {\"prev_version\": \"${GIT_TAG}\", \"base\": \"${GIT_PREV_TAG}\" }}" \
```